### ISSUE TITLE
UDS Table mappings map to non-existing profile

### ISSUE THREAD

**Issue by cooperthompson** [drajer-health/uds-plus#24]
Several UDS Data Elements in https://fhir.org/guides/hrsa/uds-plus/dataelements.html are mapped to the uds-plus-observation profile.  But that profile doesn't exist.  The possible options are:

uds-plus-lab-observation 
uds-plus-clinical-result-observation 
uds-plus-sexual-orientation-observation 
uds-plus-income-observation

**Comment by nbashyam** [comment]
Thanks Cooper, will update the build site with the right ones. It is going to be different based on Assessment vs Labs

**Comment by cooperthompson** [comment]
Do you have a CI Build where we can preview changes before they get rolled in to the next published version?

**Comment by nbashyam** [comment]
Yes, the link to the CI Build is here: https://build.fhir.org/ig/drajer-health/uds-plus/
We will start applying the changes from next week.

**Comment by anthonygalba** [comment]
It looks like this issue was fixed in IG 1.1.0, but I'm a bit confused as the [Data Elements and FHIR Mappings](https://fhir.org/guides/hrsa/uds-plus/dataelements.html) page was not updated (it still maps Laboratory Test, Performed to the generic UDS Plus Observation resource instead of the UDS Plus Lab Observation resource).
It looks like we're got three different observation resource profiles now. I just want to confirm that I have the QDM template to UDS+ FHIR resource mapping correct:
Diagnostic Study, Performed = UDS Plus Clinical Result Observation
Laboratory Test, Performed = UDS Plus Lab Observation
Assessment, Performed = UDS Plus Observation
Physical Exam, Performed = UDS Plus Observation
